includes/class-sensei-course.php

```diff
@@ -145,17 +145,87 @@ public function __construct() {
 
 		add_action( 'template_redirect', [ $this, 'setup_single_course_page' ] );
 		add_action( 'sensei_loaded', [ $this, 'add_legacy_course_hooks' ] );
+
+		// Add custom navigation.
+		add_action( 'in_admin_header', [ $this, 'add_custom_navigation' ] );
+		add_action( 'submenu_file', [ $this, 'highlight_menu_item' ] );
+	}
+
+	/**
+	 * Highlight the menu item for the course pages.
+	 *
+	 * @since 4.0.0
+	 * @access private
+	 *
+	 * @param string $submenu_file
```

Comment:
The parameters and return value would both benefit from a brief description to help future developers who touch this function.

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/4672/commits/48499f712a0b1a30929c5a7ae18744987a355a3f